packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -295,8 +283,12 @@ describeNoCompat("Named root data stores", (getTestObjectProvider) => {
         });
 
         it("Sending a bad alias message returns error", async () => {
```

Comment:
Why does this file need to change?

Reply:
I believe the tests were failing when I made some changes. They would likely pass if I revert this file, but I think this implementation is cleaner and easier to debug future test failures (previously would just say promise timeout).